pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1616,6 +1647,50 @@ func configAdapters(ctx *domainContext, config types.DomainConfig) error {
 	return nil
 }
 
+func writeMountPointsToFile(mountPoints []MountPoints, status types.DomainStatus, file *os.File) error {
+	if len(mountPoints) != len(status.DiskStatusList){
```

Comment:
Is this correct? In the case of containers, the main container image is not mounted - right?

Reply:
Corrected this condition.